tests/test_DGEList.py

```diff
@@ -203,11 +203,24 @@ def test_cpm():
     assert dge_list.counts[0][0] == first_pos * 1e6 / col_sum[0]
 
 
+def test_tpm():
+    dge_list = DGEList(filename=str(get_dataset_path(TEST_DATASET_NPZ)))
+    first_pos = dge_list.counts[0][0]
+    gene_lengths = np.random.randint(10, 1000, size=dge_list.genes.shape)
```

Comment:
If you are using `random` please set the seed first so we can ensure reproducible test results.

Reply:
Along these lines check out plugins for harnessing the power of random:

https://pypi.org/project/pytest-randomly/

Or better yet, bounded random:

https://github.com/HypothesisWorks/hypothesis/tree/master/hypothesis-python